.changeset/late-planes-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components': patch
+---
+
+Updated `<post-popover-trigger>` to provide the option to wrap the popover, as alternative to using an id reference.
```

Comment:
```suggestion
Updated the `<post-popover-trigger>` component, to provide the ability to wrap it around the `<post-popover>` element, instead of using the `id` and `for` attribute references.
```